index.js

```diff
@@ -97,6 +97,8 @@ function matter(str, options) {
   var con = str.substr(end + b.length);
   if (con.charAt(0) === '\n') {
     con = con.substr(1);
+  } else if (con.charAt(0) === '\r' && con.charAt(1) === '\n') {
+    con = con.substr(2);
   }
```

Comment:
I tried to workaround this at the metalsmith level by normalizing new lines to `\n` but the principle in metalsmith is not to do changes like this on its own (https://github.com/metalsmith/metalsmith/pull/256#discussion_r84148370).